views/find_missing.py

```diff
@@ -92,9 +114,136 @@ def render_search_button(self):
         btn_layout.addWidget(self.search_button)
         self.addLayout(btn_layout)
 
-    def search(self):
-        # Search through fetched cards here
-        deck_id = self.deck_selection_enabled and self.deck_chooser.selectedId()
+    def render_search_preview(self):
+        search_preview_layout = QHBoxLayout()
+        search_text = QLabel()
+        search_text.setText("Search:")
+        search_preview_layout.addWidget(search_text)
+        
+        self.search_preview = QTextEdit()
+        self.search_preview.setReadOnly(True)
+        self.search_preview.setFixedHeight(40)
+        search_preview_layout.addWidget(self.search_preview)
+
+        # self.copy_preview_button = QPushButton("Copy")
+        # self.copy_preview_button.clicked.connect(self.copy_preview)
+        # search_preview_layout.addWidget(self.copy_preview_button)
+
+        self.addLayout(search_preview_layout)
+
+    def render_results_area(self):
+        self.results_area = QListWidget()
+        self.addWidget(self.results_area)
+
+
+
+########################
+#######  Non-UI  #######
+########################
+
+
+
+    # def copy_preview(self):
```

Comment:
Do we want to keep this code around? Personally, I'm kinda against having commented code in the codebase; at work it has led to old stuff that gets forgotten. If we are going to use it, I'd say leave it uncommented and just don't call the function. Maybe a `#TODO`?

Reply:
Uncommented and not calling it sounds fine to me. I just know I will come back and fix this part eventually.